.github/workflows/test_functional.yml

```diff
@@ -1,8 +1,17 @@
 name: functional tests
 
 on:
-  pull_request:
   workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/*.ya?ml'
+      - '!.github/workflows/test_functional.yml'
+      - 'cylc/flow/etc/syntax/**'
+      - 'etc/syntax/**'
+      - 'tests/unit/**'
+      - 'tests/integration/**'
+      - '**.md'
+      - '**/README*/**'
```

Comment:
:+1: good idea! We need to start adding these to the other repos later too. Thanks!